PharoLauncherTutorial/pharolauncher_tutorial.pillar

```diff
@@ -0,0 +1,130 @@
+
+As Pharo is open source, it growing rapidly owing to the contributions of people all around the world. Each day we have a new update of the image of Pharo which makes it cumbersome to keep track of updates. It becomes quite a task when one has to download a new image seperately each he/she plans to work on something having the latest issues fixed, new features added etc.
+
+That's where the PharoLauncher comes in the picture.
+
+Pharo Launcher, a cross-platform application that 
+- lets you manage your Pharo images (launch, rename, copy and delete); 
+- lets you download image templates (i.e., zip archives) from many  different sources (Jenkins, files.pharo.org, and your local cache); 
+- lets you create new images from any template.
+
+The idea behind the Pharo Launcher is that you should be able to access it very rapidly from your OS application launcher. As a result launching any image is never more than 3 clicks away. 
+
+"PharoLauncher" is useful to a user who develops and needs to constantly switch between images. PharoLauncher is also a very handy tool to download specific image update versions if you want to reproduce or fix Pharo bugs.Pharo Launcher is a Pharo-based application allowing you to manage a list of images (download, rename, delete) and switch between them without aditional tools. 
+
+
+
+!! Downloading/Installing PharoLauncher
+
+!!!Linux Ubuntu:
+(*http://pharo.org/download#ubuntu*)
+
+Ubuntu users can use the dedicated ppa to install Pharo
+
+[[[
+sudo add-apt-repository ppa:pharo/stable
+sudo dpkg --add-architecture i386
+sudo apt-get update
+sudo apt-get install pharo-launcher
+]]]
+
+If you don't have the add-apt-repository command, install the software-properties-common package and try again.
+
+If you are on a server (no GUI), you can get a Pharo Virtual Machine by installing pharo-vm-core.
+
+!!!ArchLinux :
+(*http://lists.pharo.org/pipermail/pharo-users_lists.pharo.org/2014-March/010932.html*)
+[[[
+$ yaourt pharo-vm-latest
+$ pharo /path/to/your/image
+]]]
+
+There's also a pharo-launcher package that depends on pharo-vm:
+[[[
+$ yaourt pharo-launcher
+$ pharo-launcher
+]]]
+
+!!!Windows:
+Download and install the executable from the link provided here (*https://ci.inria.fr/pharo/view/Launcher/job/Launcher-Win/*)
+
+
+!!!MacOS:
+Use the link (*https://ci.inria.fr/pharo/view/Launcher/job/Launcher-Mac/*)to install PharoLauncher on Mac system.
+
+
+!!Using PharoLauncher
+
+Launch the Pharo launcher image using the platform-specific VM.
```

Comment:
This is where some more detail would help. On Mac, the Launcher in installed in /Applications. On Ubuntu, the Launcher is installed as `/usr/bin/pharo`, so you can type `pharo` on the command line to start it. I'm not sure about Windows